pbs.py

```diff
@@ -58,11 +58,12 @@ def __init__(self):
 @click.option('--server',  help='The PYBOSSA server')
 @click.option('--api-key', help='Your PYBOSSA API-KEY')
 @click.option('--all', help='Search across all projects')
+@click.option('--limit', type=int, default=0, help='Override server limit for tasks manipulation')
```

Comment:
We do we want to override sever's limit? The idea is to use what we have from the server, correct? You can tune the limits on PYBOSSA server, and this should get them. Correct?

Reply:
Yes it should take it from the server. I thought to give the ability to further limit the rate if one wants to run multiple instances in parallel.

Reply:
That should be handled automatically by pbs as we get the remaining number of requests in the headers. I would remove it to avoid "problems" and keep it as you have it, as it looks amazing.